server/eb/Makefile

```diff
@@ -27,6 +27,9 @@ build: clean
 	  mkdir -p artifact.dir/server/common/web/static/deploy; \
 	  cp -r static/* artifact.dir/server/common/web/static/deploy;  \
 	fi; \
+	if [ -d scripts ] ; then \
```

Comment:
very tiny point: should this directory be called `inline_scripts`?

Reply:
I was trying to make it a place you could put scripts or inline_scripts. 
But I don't have a strong preference.